sparse/_compressed/compressed.py

```diff
@@ -1,4 +1,5 @@
 import copy as _copy
+from numba.core.errors import CompilerError
```

Comment:
Do we need this anywhere?

Reply:
No, probably just overactive vscode autocomplete on my end.